src/common/misc/FileFunctions.h

```diff
@@ -79,7 +79,7 @@ typedef enum
     FILE_TYPE_NOT_KNOWN
 } FileType;
 
-VisItStat_t* const FILE_TYPE_DONT_STAT = ((VisItStat_t*)0x1);
```

Comment:
Was this one allowed by the compiler? It's a scary one

Reply:
yes